README.md

```diff
@@ -1,9 +1,48 @@
+#xmlseclibs 
+
 xmlseclibs is a library written in PHP for working with XML Encryption and Signatures.
 
+The author of xmlseclibs is Rob Richards.
+
+# Requirements
+
 xmlseclibs requires PHP version 5 or greater.
 
-The author of xmlseclibs is Rob Richards.
 
-xmlseclibs is being used in many different software, one of them is simpleSAMLphp. 
+## How to Install
+
+Install with [`composer.phar`][1]. 
+
+Create a `composer.json` file:
+
+```json
+{
+    "require": {
+        "robrichards/xmlseclibs": "dev-master"
```

Comment:
dev-master is a bad practice. Just remove all this example and let the composer require one